ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMap/GenerateOfflineMap.cpp

```diff
@@ -105,10 +105,10 @@ void GenerateOfflineMap::generateMapByExtent(double xCorner1, double yCorner1, d
     if (generateJob)
     {
       // connect to the job's status changed signal
-      connect(generateJob, &GenerateOfflineMapJob::statusChanged, this, [this, generateJob]()
+      connect(generateJob, &GenerateOfflineMapJob::statusChanged, this, [this, generateJob](JobStatus jobStatus)
```

Comment:
don't need to capture `generateJob` anymore

Reply:
oh, sorry my bad. you do use the job further down. that's fine then. you could move all of that completion stuff out to the `jobDone` signal but this is fine.